openapi/scim/scim.yaml

```diff
@@ -95,8 +95,11 @@ paths:
             application/scim+json:
               schema:
                 $ref: '#/components/schemas/ListResponseUsers'
-        '404':
-          description: User not found
+        '400':
+          description: |-
+            Invalid input
+            <br>Invalid Filter
```

Comment:
It should have the exact error message the api is going to provide.
Like for example if we provide filter other than userName or externalId  the error message will be `Only "userName" "externalId" filter attribute supported.`

Reply:
Changes are made.